tests/functional/builtins/folding/test_slice.py

```diff
@@ -0,0 +1,116 @@
+import string
+
+import pytest
+from hypothesis import given, settings
+from hypothesis import strategies as st
+
+from vyper import ast as vy_ast
+from vyper.builtins import functions as vy_fn
+from vyper.exceptions import ArgumentException
+
+
+@pytest.mark.fuzzing
+@settings(max_examples=50)
+@given(
+    a=st.integers(min_value=0, max_value=2**256 - 1),
+    s=st.integers(min_value=0, max_value=31),
+    le=st.integers(min_value=1, max_value=32),
+)
+def test_slice_bytes32(get_contract, a, s, le):
+    a = hex(a)
+    while len(a) < 66:
```

Comment:
we could just ask hypothesis for a string/bytes! https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.binary

Reply:
Absolutely, looks much better now, thanks !